Source/AliveLib/Well.cpp

```diff
@@ -158,6 +158,17 @@ void Well::vScreenChanged_4E3070()
     field_6_flags.Set(BaseGameObject::eDead);
 }
 
+__int16 Well_NextRandom()
+{
+    sWellRndSeed_563AA0++;
+
+    if (sWellRndSeed_563AA0 > 255)
+    {
+        sWellRndSeed_563AA0 = 1;
```

Comment:
didn't it reset to 0 not 1?

Reply:
yeah, but it still needs to be incremented by 1.